lib/styles/components/_links.scss

```diff
@@ -1,21 +0,0 @@
-@import '../settings/colors/tokens';
-
-.a-link {
```

Comment:
moved to platform, as we have it there in Storybook https://github.com/bethinkpl/wnl-platform/pull/4966/commits/c397d5eacd6332525d6abde67b1604bb9ac0066d